dashboard/app/models/plc/enrollment_unit_assignment.rb

```diff
@@ -0,0 +1,47 @@
+# == Schema Information
```

Comment:
Please add the first sentence in your note as a file comment at the head of the file.